pkg/controllers/managementuser/rbac/handler_base.go

```diff
@@ -43,6 +43,9 @@ const (
 	rtbLabelUpdated                  = "authz.cluster.cattle.io/rtb-label-updated"
 	rtbCrbRbLabelsUpdated            = "authz.cluster.cattle.io/crb-rb-labels-updated"
 	impersonationLabel               = "authz.cluster.cattle.io/impersonator"
+
+	rolesCircularSoftLimit = 100
+	rolesCircularHardLimit = 500
```

Comment:
I had thought about this, but with it being a 2 off and not being sure where a good place that was shared, I decided it was fine as it is. If you have somewhere that would be a good fit, I'm 100% on-board with reducing duplicate constants.

Reply:
I'm ok with it as is. I was originally thinking we could use this for project-wide recursion limits (maybe in settings or something), but that feels like overkill (plus this could be different in different places). 

I'll close this out.